engine/src/CalciteInterpreter.cpp

```diff
@@ -644,7 +560,7 @@ blazing_frame evaluate_split_query(std::vector<std::vector<gdf_column_cpp>> inpu
 			return child_frame;
 		} else if(is_filter(query[0])) {
 			blazing_timer.reset();  // doing a reset before to not include other calls to evaluate_split_query
-			process_filter(queryContext, child_frame, query[0]);
+			// process_filter(queryContext, child_frame, query[0]);
```

Comment:
why is this commented out? how is it done now?

Reply:
same as above